src/Protocol/Protocol18x.cpp

```diff
@@ -247,6 +257,16 @@ void cProtocol180::SendBlockChanges(int a_ChunkX, int a_ChunkZ, const sSetBlockV
 
 
 
+void cProtocol180::SendCameraSetTo(const cEntity & a_Entity)
```

Comment:
You're right. According to [wiki.vg](http://wiki.vg/Protocol#Camera) this packet is just named `Camera`. But be aware, that this packet works with **every entity**, e.g. you can also look through the "eyes" of a minecart. Maybe name it `SendSetCameraView` or `SendSetEntityView` if you find the current text misleading?